src/backend/gpopt/gpdbwrappers.cpp

```diff
@@ -2735,4 +2747,40 @@ gpdb::IsTypeRange(Oid typid)
 	return false;
 }
 
+bool
+gpdb::MarkMDCacheAsTransient(Relation index_rel)
```

Comment:
```suggestion
gpdb::SaveTransactionXminIfIndexNotUsable(Relation index_rel)
```

Reply:
I think it's worth keeping the word 'transient' in the function name so that it's better understood what this logic refers to.